--- thrift-0.9.1/compiler/cpp/Makefile.in	2016-10-12 11:08:49.012927396 -0600
+++ thrift-0.9.1/compiler/cpp/Makefile.in.fix	2016-10-12 11:11:25.600812386 -0600
@@ -96,27 +96,10 @@
 PROGRAMS = $(bin_PROGRAMS)
 am_thrift_OBJECTS = thrift-main.$(OBJEXT) thrift-md5.$(OBJEXT) \
 	thrift-t_generator.$(OBJEXT) thrift-parse.$(OBJEXT) \
-	thrift-t_c_glib_generator.$(OBJEXT) \
-	thrift-t_cpp_generator.$(OBJEXT) \
 	thrift-t_java_generator.$(OBJEXT) \
-	thrift-t_as3_generator.$(OBJEXT) \
-	thrift-t_csharp_generator.$(OBJEXT) \
 	thrift-t_py_generator.$(OBJEXT) \
-	thrift-t_rb_generator.$(OBJEXT) \
-	thrift-t_perl_generator.$(OBJEXT) \
-	thrift-t_php_generator.$(OBJEXT) \
-	thrift-t_erl_generator.$(OBJEXT) \
-	thrift-t_cocoa_generator.$(OBJEXT) \
-	thrift-t_st_generator.$(OBJEXT) \
-	thrift-t_ocaml_generator.$(OBJEXT) \
-	thrift-t_hs_generator.$(OBJEXT) \
-	thrift-t_xsd_generator.$(OBJEXT) \
 	thrift-t_html_generator.$(OBJEXT) \
-	thrift-t_js_generator.$(OBJEXT) \
-	thrift-t_javame_generator.$(OBJEXT) \
-	thrift-t_delphi_generator.$(OBJEXT) \
-	thrift-t_go_generator.$(OBJEXT) \
-	thrift-t_gv_generator.$(OBJEXT) thrift-t_d_generator.$(OBJEXT)
+	thrift-t_js_generator.$(OBJEXT)
 thrift_OBJECTS = $(am_thrift_OBJECTS)
 thrift_DEPENDENCIES = libparse.a
 thrift_LINK = $(LIBTOOL) --tag=CXX $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) \
@@ -377,25 +360,11 @@
 	src/parse/t_const_value.h src/parse/parse.cc \
 	src/generate/t_generator.h src/generate/t_oop_generator.h \
 	src/generate/t_html_generator.h src/windows/config.h \
-	src/windows/version.h src/generate/t_c_glib_generator.cc \
-	src/generate/t_cpp_generator.cc \
+	src/windows/version.h \
 	src/generate/t_java_generator.cc \
-	src/generate/t_as3_generator.cc \
-	src/generate/t_csharp_generator.cc \
-	src/generate/t_py_generator.cc src/generate/t_rb_generator.cc \
-	src/generate/t_perl_generator.cc \
-	src/generate/t_php_generator.cc \
-	src/generate/t_erl_generator.cc \
-	src/generate/t_cocoa_generator.cc \
-	src/generate/t_st_generator.cc \
-	src/generate/t_ocaml_generator.cc \
-	src/generate/t_hs_generator.cc src/generate/t_xsd_generator.cc \
+	src/generate/t_py_generator.cc \
 	src/generate/t_html_generator.cc \
-	src/generate/t_js_generator.cc \
-	src/generate/t_javame_generator.cc \
-	src/generate/t_delphi_generator.cc \
-	src/generate/t_go_generator.cc src/generate/t_gv_generator.cc \
-	src/generate/t_d_generator.cc
+	src/generate/t_js_generator.cc
 thrift_CPPFLAGS = -I$(srcdir)/src
 thrift_CXXFLAGS = -Wall
 thrift_LDADD = @LEXLIB@ libparse.a
@@ -518,29 +487,11 @@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-main.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-md5.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-parse.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_as3_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_c_glib_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_cocoa_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_cpp_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_csharp_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_d_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_delphi_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_erl_generator.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_go_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_gv_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_hs_generator.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_html_generator.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_java_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_javame_generator.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_js_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_ocaml_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_perl_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_php_generator.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_py_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_rb_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_st_generator.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/thrift-t_xsd_generator.Po@am__quote@
 
 .c.o:
 @am__fastdepCC_TRUE@	$(COMPILE) -MT $@ -MD -MP -MF $(DEPDIR)/$*.Tpo -c -o $@ $<
@@ -668,34 +619,6 @@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
 @am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-parse.obj `if test -f 'src/parse/parse.cc'; then $(CYGPATH_W) 'src/parse/parse.cc'; else $(CYGPATH_W) '$(srcdir)/src/parse/parse.cc'; fi`
 
-thrift-t_c_glib_generator.o: src/generate/t_c_glib_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_c_glib_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_c_glib_generator.Tpo -c -o thrift-t_c_glib_generator.o `test -f 'src/generate/t_c_glib_generator.cc' || echo '$(srcdir)/'`src/generate/t_c_glib_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_c_glib_generator.Tpo $(DEPDIR)/thrift-t_c_glib_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_c_glib_generator.cc' object='thrift-t_c_glib_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_c_glib_generator.o `test -f 'src/generate/t_c_glib_generator.cc' || echo '$(srcdir)/'`src/generate/t_c_glib_generator.cc
-
-thrift-t_c_glib_generator.obj: src/generate/t_c_glib_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_c_glib_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_c_glib_generator.Tpo -c -o thrift-t_c_glib_generator.obj `if test -f 'src/generate/t_c_glib_generator.cc'; then $(CYGPATH_W) 'src/generate/t_c_glib_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_c_glib_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_c_glib_generator.Tpo $(DEPDIR)/thrift-t_c_glib_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_c_glib_generator.cc' object='thrift-t_c_glib_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_c_glib_generator.obj `if test -f 'src/generate/t_c_glib_generator.cc'; then $(CYGPATH_W) 'src/generate/t_c_glib_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_c_glib_generator.cc'; fi`
-
-thrift-t_cpp_generator.o: src/generate/t_cpp_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_cpp_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_cpp_generator.Tpo -c -o thrift-t_cpp_generator.o `test -f 'src/generate/t_cpp_generator.cc' || echo '$(srcdir)/'`src/generate/t_cpp_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_cpp_generator.Tpo $(DEPDIR)/thrift-t_cpp_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_cpp_generator.cc' object='thrift-t_cpp_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_cpp_generator.o `test -f 'src/generate/t_cpp_generator.cc' || echo '$(srcdir)/'`src/generate/t_cpp_generator.cc
-
-thrift-t_cpp_generator.obj: src/generate/t_cpp_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_cpp_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_cpp_generator.Tpo -c -o thrift-t_cpp_generator.obj `if test -f 'src/generate/t_cpp_generator.cc'; then $(CYGPATH_W) 'src/generate/t_cpp_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_cpp_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_cpp_generator.Tpo $(DEPDIR)/thrift-t_cpp_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_cpp_generator.cc' object='thrift-t_cpp_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_cpp_generator.obj `if test -f 'src/generate/t_cpp_generator.cc'; then $(CYGPATH_W) 'src/generate/t_cpp_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_cpp_generator.cc'; fi`
-
 thrift-t_java_generator.o: src/generate/t_java_generator.cc
 @am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_java_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_java_generator.Tpo -c -o thrift-t_java_generator.o `test -f 'src/generate/t_java_generator.cc' || echo '$(srcdir)/'`src/generate/t_java_generator.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_java_generator.Tpo $(DEPDIR)/thrift-t_java_generator.Po
@@ -710,34 +633,6 @@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
 @am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_java_generator.obj `if test -f 'src/generate/t_java_generator.cc'; then $(CYGPATH_W) 'src/generate/t_java_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_java_generator.cc'; fi`
 
-thrift-t_as3_generator.o: src/generate/t_as3_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_as3_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_as3_generator.Tpo -c -o thrift-t_as3_generator.o `test -f 'src/generate/t_as3_generator.cc' || echo '$(srcdir)/'`src/generate/t_as3_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_as3_generator.Tpo $(DEPDIR)/thrift-t_as3_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_as3_generator.cc' object='thrift-t_as3_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_as3_generator.o `test -f 'src/generate/t_as3_generator.cc' || echo '$(srcdir)/'`src/generate/t_as3_generator.cc
-
-thrift-t_as3_generator.obj: src/generate/t_as3_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_as3_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_as3_generator.Tpo -c -o thrift-t_as3_generator.obj `if test -f 'src/generate/t_as3_generator.cc'; then $(CYGPATH_W) 'src/generate/t_as3_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_as3_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_as3_generator.Tpo $(DEPDIR)/thrift-t_as3_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_as3_generator.cc' object='thrift-t_as3_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_as3_generator.obj `if test -f 'src/generate/t_as3_generator.cc'; then $(CYGPATH_W) 'src/generate/t_as3_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_as3_generator.cc'; fi`
-
-thrift-t_csharp_generator.o: src/generate/t_csharp_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_csharp_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_csharp_generator.Tpo -c -o thrift-t_csharp_generator.o `test -f 'src/generate/t_csharp_generator.cc' || echo '$(srcdir)/'`src/generate/t_csharp_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_csharp_generator.Tpo $(DEPDIR)/thrift-t_csharp_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_csharp_generator.cc' object='thrift-t_csharp_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_csharp_generator.o `test -f 'src/generate/t_csharp_generator.cc' || echo '$(srcdir)/'`src/generate/t_csharp_generator.cc
-
-thrift-t_csharp_generator.obj: src/generate/t_csharp_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_csharp_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_csharp_generator.Tpo -c -o thrift-t_csharp_generator.obj `if test -f 'src/generate/t_csharp_generator.cc'; then $(CYGPATH_W) 'src/generate/t_csharp_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_csharp_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_csharp_generator.Tpo $(DEPDIR)/thrift-t_csharp_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_csharp_generator.cc' object='thrift-t_csharp_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_csharp_generator.obj `if test -f 'src/generate/t_csharp_generator.cc'; then $(CYGPATH_W) 'src/generate/t_csharp_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_csharp_generator.cc'; fi`
-
 thrift-t_py_generator.o: src/generate/t_py_generator.cc
 @am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_py_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_py_generator.Tpo -c -o thrift-t_py_generator.o `test -f 'src/generate/t_py_generator.cc' || echo '$(srcdir)/'`src/generate/t_py_generator.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_py_generator.Tpo $(DEPDIR)/thrift-t_py_generator.Po
@@ -752,132 +647,6 @@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
 @am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_py_generator.obj `if test -f 'src/generate/t_py_generator.cc'; then $(CYGPATH_W) 'src/generate/t_py_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_py_generator.cc'; fi`
 
-thrift-t_rb_generator.o: src/generate/t_rb_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_rb_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_rb_generator.Tpo -c -o thrift-t_rb_generator.o `test -f 'src/generate/t_rb_generator.cc' || echo '$(srcdir)/'`src/generate/t_rb_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_rb_generator.Tpo $(DEPDIR)/thrift-t_rb_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_rb_generator.cc' object='thrift-t_rb_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_rb_generator.o `test -f 'src/generate/t_rb_generator.cc' || echo '$(srcdir)/'`src/generate/t_rb_generator.cc
-
-thrift-t_rb_generator.obj: src/generate/t_rb_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_rb_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_rb_generator.Tpo -c -o thrift-t_rb_generator.obj `if test -f 'src/generate/t_rb_generator.cc'; then $(CYGPATH_W) 'src/generate/t_rb_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_rb_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_rb_generator.Tpo $(DEPDIR)/thrift-t_rb_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_rb_generator.cc' object='thrift-t_rb_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_rb_generator.obj `if test -f 'src/generate/t_rb_generator.cc'; then $(CYGPATH_W) 'src/generate/t_rb_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_rb_generator.cc'; fi`
-
-thrift-t_perl_generator.o: src/generate/t_perl_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_perl_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_perl_generator.Tpo -c -o thrift-t_perl_generator.o `test -f 'src/generate/t_perl_generator.cc' || echo '$(srcdir)/'`src/generate/t_perl_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_perl_generator.Tpo $(DEPDIR)/thrift-t_perl_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_perl_generator.cc' object='thrift-t_perl_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_perl_generator.o `test -f 'src/generate/t_perl_generator.cc' || echo '$(srcdir)/'`src/generate/t_perl_generator.cc
-
-thrift-t_perl_generator.obj: src/generate/t_perl_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_perl_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_perl_generator.Tpo -c -o thrift-t_perl_generator.obj `if test -f 'src/generate/t_perl_generator.cc'; then $(CYGPATH_W) 'src/generate/t_perl_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_perl_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_perl_generator.Tpo $(DEPDIR)/thrift-t_perl_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_perl_generator.cc' object='thrift-t_perl_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_perl_generator.obj `if test -f 'src/generate/t_perl_generator.cc'; then $(CYGPATH_W) 'src/generate/t_perl_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_perl_generator.cc'; fi`
-
-thrift-t_php_generator.o: src/generate/t_php_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_php_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_php_generator.Tpo -c -o thrift-t_php_generator.o `test -f 'src/generate/t_php_generator.cc' || echo '$(srcdir)/'`src/generate/t_php_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_php_generator.Tpo $(DEPDIR)/thrift-t_php_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_php_generator.cc' object='thrift-t_php_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_php_generator.o `test -f 'src/generate/t_php_generator.cc' || echo '$(srcdir)/'`src/generate/t_php_generator.cc
-
-thrift-t_php_generator.obj: src/generate/t_php_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_php_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_php_generator.Tpo -c -o thrift-t_php_generator.obj `if test -f 'src/generate/t_php_generator.cc'; then $(CYGPATH_W) 'src/generate/t_php_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_php_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_php_generator.Tpo $(DEPDIR)/thrift-t_php_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_php_generator.cc' object='thrift-t_php_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_php_generator.obj `if test -f 'src/generate/t_php_generator.cc'; then $(CYGPATH_W) 'src/generate/t_php_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_php_generator.cc'; fi`
-
-thrift-t_erl_generator.o: src/generate/t_erl_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_erl_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_erl_generator.Tpo -c -o thrift-t_erl_generator.o `test -f 'src/generate/t_erl_generator.cc' || echo '$(srcdir)/'`src/generate/t_erl_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_erl_generator.Tpo $(DEPDIR)/thrift-t_erl_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_erl_generator.cc' object='thrift-t_erl_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_erl_generator.o `test -f 'src/generate/t_erl_generator.cc' || echo '$(srcdir)/'`src/generate/t_erl_generator.cc
-
-thrift-t_erl_generator.obj: src/generate/t_erl_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_erl_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_erl_generator.Tpo -c -o thrift-t_erl_generator.obj `if test -f 'src/generate/t_erl_generator.cc'; then $(CYGPATH_W) 'src/generate/t_erl_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_erl_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_erl_generator.Tpo $(DEPDIR)/thrift-t_erl_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_erl_generator.cc' object='thrift-t_erl_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_erl_generator.obj `if test -f 'src/generate/t_erl_generator.cc'; then $(CYGPATH_W) 'src/generate/t_erl_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_erl_generator.cc'; fi`
-
-thrift-t_cocoa_generator.o: src/generate/t_cocoa_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_cocoa_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_cocoa_generator.Tpo -c -o thrift-t_cocoa_generator.o `test -f 'src/generate/t_cocoa_generator.cc' || echo '$(srcdir)/'`src/generate/t_cocoa_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_cocoa_generator.Tpo $(DEPDIR)/thrift-t_cocoa_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_cocoa_generator.cc' object='thrift-t_cocoa_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_cocoa_generator.o `test -f 'src/generate/t_cocoa_generator.cc' || echo '$(srcdir)/'`src/generate/t_cocoa_generator.cc
-
-thrift-t_cocoa_generator.obj: src/generate/t_cocoa_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_cocoa_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_cocoa_generator.Tpo -c -o thrift-t_cocoa_generator.obj `if test -f 'src/generate/t_cocoa_generator.cc'; then $(CYGPATH_W) 'src/generate/t_cocoa_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_cocoa_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_cocoa_generator.Tpo $(DEPDIR)/thrift-t_cocoa_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_cocoa_generator.cc' object='thrift-t_cocoa_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_cocoa_generator.obj `if test -f 'src/generate/t_cocoa_generator.cc'; then $(CYGPATH_W) 'src/generate/t_cocoa_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_cocoa_generator.cc'; fi`
-
-thrift-t_st_generator.o: src/generate/t_st_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_st_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_st_generator.Tpo -c -o thrift-t_st_generator.o `test -f 'src/generate/t_st_generator.cc' || echo '$(srcdir)/'`src/generate/t_st_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_st_generator.Tpo $(DEPDIR)/thrift-t_st_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_st_generator.cc' object='thrift-t_st_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_st_generator.o `test -f 'src/generate/t_st_generator.cc' || echo '$(srcdir)/'`src/generate/t_st_generator.cc
-
-thrift-t_st_generator.obj: src/generate/t_st_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_st_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_st_generator.Tpo -c -o thrift-t_st_generator.obj `if test -f 'src/generate/t_st_generator.cc'; then $(CYGPATH_W) 'src/generate/t_st_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_st_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_st_generator.Tpo $(DEPDIR)/thrift-t_st_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_st_generator.cc' object='thrift-t_st_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_st_generator.obj `if test -f 'src/generate/t_st_generator.cc'; then $(CYGPATH_W) 'src/generate/t_st_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_st_generator.cc'; fi`
-
-thrift-t_ocaml_generator.o: src/generate/t_ocaml_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_ocaml_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_ocaml_generator.Tpo -c -o thrift-t_ocaml_generator.o `test -f 'src/generate/t_ocaml_generator.cc' || echo '$(srcdir)/'`src/generate/t_ocaml_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_ocaml_generator.Tpo $(DEPDIR)/thrift-t_ocaml_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_ocaml_generator.cc' object='thrift-t_ocaml_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_ocaml_generator.o `test -f 'src/generate/t_ocaml_generator.cc' || echo '$(srcdir)/'`src/generate/t_ocaml_generator.cc
-
-thrift-t_ocaml_generator.obj: src/generate/t_ocaml_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_ocaml_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_ocaml_generator.Tpo -c -o thrift-t_ocaml_generator.obj `if test -f 'src/generate/t_ocaml_generator.cc'; then $(CYGPATH_W) 'src/generate/t_ocaml_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_ocaml_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_ocaml_generator.Tpo $(DEPDIR)/thrift-t_ocaml_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_ocaml_generator.cc' object='thrift-t_ocaml_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_ocaml_generator.obj `if test -f 'src/generate/t_ocaml_generator.cc'; then $(CYGPATH_W) 'src/generate/t_ocaml_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_ocaml_generator.cc'; fi`
-
-thrift-t_hs_generator.o: src/generate/t_hs_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_hs_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_hs_generator.Tpo -c -o thrift-t_hs_generator.o `test -f 'src/generate/t_hs_generator.cc' || echo '$(srcdir)/'`src/generate/t_hs_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_hs_generator.Tpo $(DEPDIR)/thrift-t_hs_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_hs_generator.cc' object='thrift-t_hs_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_hs_generator.o `test -f 'src/generate/t_hs_generator.cc' || echo '$(srcdir)/'`src/generate/t_hs_generator.cc
-
-thrift-t_hs_generator.obj: src/generate/t_hs_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_hs_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_hs_generator.Tpo -c -o thrift-t_hs_generator.obj `if test -f 'src/generate/t_hs_generator.cc'; then $(CYGPATH_W) 'src/generate/t_hs_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_hs_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_hs_generator.Tpo $(DEPDIR)/thrift-t_hs_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_hs_generator.cc' object='thrift-t_hs_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_hs_generator.obj `if test -f 'src/generate/t_hs_generator.cc'; then $(CYGPATH_W) 'src/generate/t_hs_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_hs_generator.cc'; fi`
-
-thrift-t_xsd_generator.o: src/generate/t_xsd_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_xsd_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_xsd_generator.Tpo -c -o thrift-t_xsd_generator.o `test -f 'src/generate/t_xsd_generator.cc' || echo '$(srcdir)/'`src/generate/t_xsd_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_xsd_generator.Tpo $(DEPDIR)/thrift-t_xsd_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_xsd_generator.cc' object='thrift-t_xsd_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_xsd_generator.o `test -f 'src/generate/t_xsd_generator.cc' || echo '$(srcdir)/'`src/generate/t_xsd_generator.cc
-
-thrift-t_xsd_generator.obj: src/generate/t_xsd_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_xsd_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_xsd_generator.Tpo -c -o thrift-t_xsd_generator.obj `if test -f 'src/generate/t_xsd_generator.cc'; then $(CYGPATH_W) 'src/generate/t_xsd_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_xsd_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_xsd_generator.Tpo $(DEPDIR)/thrift-t_xsd_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_xsd_generator.cc' object='thrift-t_xsd_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_xsd_generator.obj `if test -f 'src/generate/t_xsd_generator.cc'; then $(CYGPATH_W) 'src/generate/t_xsd_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_xsd_generator.cc'; fi`
-
 thrift-t_html_generator.o: src/generate/t_html_generator.cc
 @am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_html_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_html_generator.Tpo -c -o thrift-t_html_generator.o `test -f 'src/generate/t_html_generator.cc' || echo '$(srcdir)/'`src/generate/t_html_generator.cc
 @am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_html_generator.Tpo $(DEPDIR)/thrift-t_html_generator.Po
@@ -906,76 +675,6 @@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
 @am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_js_generator.obj `if test -f 'src/generate/t_js_generator.cc'; then $(CYGPATH_W) 'src/generate/t_js_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_js_generator.cc'; fi`
 
-thrift-t_javame_generator.o: src/generate/t_javame_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_javame_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_javame_generator.Tpo -c -o thrift-t_javame_generator.o `test -f 'src/generate/t_javame_generator.cc' || echo '$(srcdir)/'`src/generate/t_javame_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_javame_generator.Tpo $(DEPDIR)/thrift-t_javame_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_javame_generator.cc' object='thrift-t_javame_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_javame_generator.o `test -f 'src/generate/t_javame_generator.cc' || echo '$(srcdir)/'`src/generate/t_javame_generator.cc
-
-thrift-t_javame_generator.obj: src/generate/t_javame_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_javame_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_javame_generator.Tpo -c -o thrift-t_javame_generator.obj `if test -f 'src/generate/t_javame_generator.cc'; then $(CYGPATH_W) 'src/generate/t_javame_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_javame_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_javame_generator.Tpo $(DEPDIR)/thrift-t_javame_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_javame_generator.cc' object='thrift-t_javame_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_javame_generator.obj `if test -f 'src/generate/t_javame_generator.cc'; then $(CYGPATH_W) 'src/generate/t_javame_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_javame_generator.cc'; fi`
-
-thrift-t_delphi_generator.o: src/generate/t_delphi_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_delphi_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_delphi_generator.Tpo -c -o thrift-t_delphi_generator.o `test -f 'src/generate/t_delphi_generator.cc' || echo '$(srcdir)/'`src/generate/t_delphi_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_delphi_generator.Tpo $(DEPDIR)/thrift-t_delphi_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_delphi_generator.cc' object='thrift-t_delphi_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_delphi_generator.o `test -f 'src/generate/t_delphi_generator.cc' || echo '$(srcdir)/'`src/generate/t_delphi_generator.cc
-
-thrift-t_delphi_generator.obj: src/generate/t_delphi_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_delphi_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_delphi_generator.Tpo -c -o thrift-t_delphi_generator.obj `if test -f 'src/generate/t_delphi_generator.cc'; then $(CYGPATH_W) 'src/generate/t_delphi_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_delphi_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_delphi_generator.Tpo $(DEPDIR)/thrift-t_delphi_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_delphi_generator.cc' object='thrift-t_delphi_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_delphi_generator.obj `if test -f 'src/generate/t_delphi_generator.cc'; then $(CYGPATH_W) 'src/generate/t_delphi_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_delphi_generator.cc'; fi`
-
-thrift-t_go_generator.o: src/generate/t_go_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_go_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_go_generator.Tpo -c -o thrift-t_go_generator.o `test -f 'src/generate/t_go_generator.cc' || echo '$(srcdir)/'`src/generate/t_go_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_go_generator.Tpo $(DEPDIR)/thrift-t_go_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_go_generator.cc' object='thrift-t_go_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_go_generator.o `test -f 'src/generate/t_go_generator.cc' || echo '$(srcdir)/'`src/generate/t_go_generator.cc
-
-thrift-t_go_generator.obj: src/generate/t_go_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_go_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_go_generator.Tpo -c -o thrift-t_go_generator.obj `if test -f 'src/generate/t_go_generator.cc'; then $(CYGPATH_W) 'src/generate/t_go_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_go_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_go_generator.Tpo $(DEPDIR)/thrift-t_go_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_go_generator.cc' object='thrift-t_go_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_go_generator.obj `if test -f 'src/generate/t_go_generator.cc'; then $(CYGPATH_W) 'src/generate/t_go_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_go_generator.cc'; fi`
-
-thrift-t_gv_generator.o: src/generate/t_gv_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_gv_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_gv_generator.Tpo -c -o thrift-t_gv_generator.o `test -f 'src/generate/t_gv_generator.cc' || echo '$(srcdir)/'`src/generate/t_gv_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_gv_generator.Tpo $(DEPDIR)/thrift-t_gv_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_gv_generator.cc' object='thrift-t_gv_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_gv_generator.o `test -f 'src/generate/t_gv_generator.cc' || echo '$(srcdir)/'`src/generate/t_gv_generator.cc
-
-thrift-t_gv_generator.obj: src/generate/t_gv_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_gv_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_gv_generator.Tpo -c -o thrift-t_gv_generator.obj `if test -f 'src/generate/t_gv_generator.cc'; then $(CYGPATH_W) 'src/generate/t_gv_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_gv_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_gv_generator.Tpo $(DEPDIR)/thrift-t_gv_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_gv_generator.cc' object='thrift-t_gv_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_gv_generator.obj `if test -f 'src/generate/t_gv_generator.cc'; then $(CYGPATH_W) 'src/generate/t_gv_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_gv_generator.cc'; fi`
-
-thrift-t_d_generator.o: src/generate/t_d_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_d_generator.o -MD -MP -MF $(DEPDIR)/thrift-t_d_generator.Tpo -c -o thrift-t_d_generator.o `test -f 'src/generate/t_d_generator.cc' || echo '$(srcdir)/'`src/generate/t_d_generator.cc
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_d_generator.Tpo $(DEPDIR)/thrift-t_d_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_d_generator.cc' object='thrift-t_d_generator.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_d_generator.o `test -f 'src/generate/t_d_generator.cc' || echo '$(srcdir)/'`src/generate/t_d_generator.cc
-
-thrift-t_d_generator.obj: src/generate/t_d_generator.cc
-@am__fastdepCXX_TRUE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -MT thrift-t_d_generator.obj -MD -MP -MF $(DEPDIR)/thrift-t_d_generator.Tpo -c -o thrift-t_d_generator.obj `if test -f 'src/generate/t_d_generator.cc'; then $(CYGPATH_W) 'src/generate/t_d_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_d_generator.cc'; fi`
-@am__fastdepCXX_TRUE@	$(am__mv) $(DEPDIR)/thrift-t_d_generator.Tpo $(DEPDIR)/thrift-t_d_generator.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	source='src/generate/t_d_generator.cc' object='thrift-t_d_generator.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(thrift_CPPFLAGS) $(CPPFLAGS) $(thrift_CXXFLAGS) $(CXXFLAGS) -c -o thrift-t_d_generator.obj `if test -f 'src/generate/t_d_generator.cc'; then $(CYGPATH_W) 'src/generate/t_d_generator.cc'; else $(CYGPATH_W) '$(srcdir)/src/generate/t_d_generator.cc'; fi`
-
 .ll.cc:
 	$(am__skiplex) $(SHELL) $(YLWRAP) $< $(LEX_OUTPUT_ROOT).c $@ -- $(LEXCOMPILE)
 
