--- thrift-0.9.1/compiler/cpp/src/generate/t_java_generator.cc	2016-07-01 12:44:02.556421907 -0600
+++ thrift-0.9.1/compiler/cpp/src/generate/t_java_generator.cc.fix	2016-07-01 12:41:46.798444685 -0600
@@ -2827,7 +2827,10 @@
      bool first = true;
      if (xceptions.size() > 0) {
     	 for (x_iter = xceptions.begin(); x_iter != xceptions.end(); ++x_iter) {
-    		 first ? first = false : indent(f_service_) << "else ";
+		 if (first) {
+			 indent(f_service_) << "else ";
+		 }
+    		 first = false;
     		 indent(f_service_) << "if (e instanceof " << type_name((*x_iter)->get_type(), false, false)<<") {" << endl;
     		 indent(f_service_) << indent() << "result." << (*x_iter)->get_name() << " = (" << type_name((*x_iter)->get_type(), false, false) << ") e;" << endl;
     	  	 indent(f_service_) << indent() << "result.set" << get_cap_name((*x_iter)->get_name()) << get_cap_name("isSet") << "(true);" << endl;
